pkg/cmd/server/server.go

```diff
@@ -439,8 +463,10 @@ func (s *server) validateBackupStorageLocations() error {
 	pluginManager := clientmgmt.NewManager(s.logger, s.logLevel, s.pluginRegistry)
 	defer pluginManager.CleanupClients()
 
-	locations, err := s.veleroClient.VeleroV1().BackupStorageLocations(s.namespace).List(metav1.ListOptions{})
-	if err != nil {
+	locations := &velerov1api.BackupStorageLocationList{}
+	if err := s.mgr.GetAPIReader().List(context.Background(), locations, &kbclient.ListOptions{
```

Comment:
Is there a reason why we are not using `GetClient`? The `GetAPIReader` talks directly to the API server and does not use the cache.

Reply:
Yes, at this point the cache hasn't been started yet.

Reply:
suggest adding a comment to that effect.

Reply:
👍 to a comment for explanation on `GetAPIReader`. We went over this on a call, but I'm likely to forget in a week 😄

Reply:
Done.